.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,17 @@
+name: pulL_request
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Convert yaml to json
+        run: python .github/workflows/lib/yaml_to_json.py resources/referers.yml referers.json
+      - name: Check
+        run: cat referers.json
```

Comment:
Please can you explain more how this github action works?  I can see you cat the output file, but how does that create an asset which can be downloaded?